index.rst

```diff
@@ -0,0 +1,267 @@
+     :name: figure_4
+     :target: _images/figure_4.png
+
+     Parameterizations of the form 0nn0.  Full is 0220
+
+.. literalinclude:: /_static/table_1.txt
+
+Things appear to settle around 0440 or 0550.  But none of these differ from the reference model by 3 sigma.
+
+However, note the similar comparisons with inner and outer set to 2 or 3. The primary and wings settings seem to settle by order 4 with higher order in the inner and outer components.
+
+
+  .. figure:: /_static/figure_5.png
+     :name: figure_5
+     :target: _images/figure_5.png
+
+     Parameterizations of the form 1nn1.
+
+  .. figure:: /_static/figure_6.png
+     :name: figure_6
+     :target: _images/figure_6.png
+
+     Parameterizations of the form 2nn2.
+
+  .. figure:: /_static/figure_7.png
+     :name: figure_7
+     :target: _images/figure_7.png
+
+     Parameterizations of the form 3nn3.
+
+Here is a full set of comparisons against 3773 where primary and wings are the same, and inner and outer are the same.  They suggest that order 3 may be nearly as good as the higher orders, with improvement in the Psf modeling appearing to max out between order 3 and 4.
+
+.. literalinclude:: /_static/table_2.txt
+
+Conclusion:
+----------
+
+It is hard to draw too many hard conclusions from these studies. For one thing, the models are close enough to each other above order 3 that the accuracy of our error measurement becomes very important. If the 2.2 multiplier which we applied (consistent with the bootstrap resampling) is correct, then orders above 4 not a useful improvement with the 2 million galaxies in this test.
+
+However, it does appear that the order 2 parameterizations are significantly different from 3773, and that order 3 is needed at least in the primary and wings modelling. But there isn't all that much to choose from above order 3.  In fact, 2332 seems to be quite close to 3773, and 3333 is even a little better.
+
+It is obvious that a lot depends on our error assumptions. In future studies, we should create several populations of the same setup to make true variation easier to determine.  Varying the Psf from point-to-point on the focal plane may also have an effect which we have not studied in this set of tests.
+
+Testing CModel Configurations
+========================
+Our earliest tests were tests of the CModel algorithm, leaving the SPA for Psf estimation at its defaults.  CModel is a shape measurement algorithm in the meas_modelfit package of the DM Stack. This study was done partly to see the effects varying the CModel task config, and partly to determine how large a galaxy sample was required.
+
+As a result of these these tests, we decided to move our tests for single host, multi-core machines at Davis to the computing farm at SLAC. 
+
+Test setup
+--------------
+Subfields: 1000 galaxies/subfield in 500 pairs, all with the same applied shear and shear angle
+           The applied shear was varied in 6 steps from .0001 to .05.
```

Comment:
Indent is doing strange things here too, and for some reason one of these lines appears in bold italics for me (using the GitHub renderer, though, so if it's not present in the LSST renderer don't worry about it).